### PR TITLE
2.1 release notes

### PR DESCRIPTION
**Pull request by pmatulis**
This is the first 2.1 bits going into master.

Original: [http://maas.ubuntu.com/docs2.1/changelog.html](http://maas.ubuntu.com/docs2.1/changelog.html)

### CODE REVIEW

File: en/2.1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+networks:
+
+- **On-demand**: Administrators can choose to map their subnet using an action on
+  the subnet details page. This action will scan the subnet just once, so that
+  observed devices on that subnet may quickly be seen in the dashboard. This
+  feature is useful after initially installing MAAS, to quickly populate the list
+  of discoveries with active devices on the network.
+
+- **Periodically** (recommended): By enabling active discovery on a per-subnet
+  basis, subnets will be scanned at a user-specified interval. (default is
```
Comment (degville) [inline en/2.1-release-notes.md]: period/full stop before the bracket should probably be removed.

File: en/2.1-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+### New HWE and GA kernel naming convention
+
+Starting with MAAS 2.1 and Ubuntu 16.04 LTS (Xenial), MAAS is adhering to a
+new naming convention for hardware enablement (HWE) kernels and general
+availability (GA) kernels. MAAS will no longer support the old naming
+convention for HWE kernels (e.g. hwe-y) when selecting images (Xenial and
+newer) to import. The new nomenclature is as follows:
+
+- `ga-<version>` : The GA kernel has the major kernel version of the kernel which
```
Comment (degville) [inline en/2.1-release-notes.md]: Should we remove the spaces before the colons (and throughout the remainder of the doc)?

Comment (pmatulis) [inline en/2.1-release-notes.md]: I did this for legibility. I'll remove them and see what it looks like.

**Comment by pmatulis** [comment]
Edits made @degville

**Comment by degville** [comment]
Brilliant - thank you!

**Pull request merged**